src/tests/cli_tests.py

```diff
@@ -1200,6 +1200,90 @@ def test_userid_escape(self):
             raise_err('userids from rnpkeys and gpg don\'t match')
         clear_keyrings()
 
+    def test_key_revoke(self):
+        clear_keyrings()
+        # Import Alice's public key and be unable to revoke
+        ret, _, _ = run_proc(RNPK, ['--homedir', RNPDIR, '--import', data_path('test_key_validity/alice-pub.asc')])
+        if ret != 0:
+            raise_err('Alice key import failed')
+        ret, out, err = run_proc(RNPK, ['--homedir', RNPDIR, '--revoke-key', 'alice'])
+        if (ret == 0) or (len(out) > 0) or not re.match(r'(?s)^.*Revoker secret key not found.*Failed to revoke a key.*', err):
+            raise_err('Wrong revocation output')
+        # Import Alice's secret key and subkey
+        ret, _, _ = run_proc(RNPK, ['--homedir', RNPDIR, '--import', data_path('test_key_validity/alice-sub-sec.pgp')])
+        if ret != 0:
+            raise_err('Alice secret key import failed')
+        # Attempt to revoke without specifying a key
+        pipe = pswd_pipe(PASSWORD)
+        ret, out, err = run_proc(RNPK, ['--homedir', RNPDIR, '--revoke', '--pass-fd', str(pipe)])
```

Comment:
@ni4 Is "revoke" a synonym for "revoke-key"? I can't find it in the map, nor in the usage text.

Reply:
@rrrooommmaaa getopt_long() accepts abbreviation of the long command instead of the whole one, i.e. `--revok` will work as well. This doesn't seem to be documented in manual pages but can be seen from practice or glibc sources :)
Anyway, this line most likely will be subject to update once we have --revoke-cert/--revoke-userid.